robot_sf/render/sim_view.py

```
@@ -74,62 +74,135 @@ class SimulationView:
+
+        # information
+        self._add_text(state.timestep, state)
+        if self.display_help:
+            self._add_help_text()
+
+        # update the display
+        pygame.display.update()
+
+    def handle_events(self):
+        """Handle PyGame events on the main thread"""
+        if threading.current_thread() is not self._main_thread:
```

Ruff

Undefined name `threading`

(F821)